src/netex_france/stops.rs

```diff
@@ -312,6 +321,61 @@ impl<'a> StopExporter<'a> {
         Ok(Some(centroid))
     }
 
+    fn generate_quay_accessibility(&self, equipment_id: Option<&str>) -> Option<Element> {
```

Comment:
This is a reference onto the model. So you can make it explicit with the following lifetime.
```suggestion
    fn generate_quay_accessibility(&self, equipment_id: Option<&'a str>) -> Option<Element> {
```

And maybe in this case, it'd be easier to have an `Option<&'a String>`. It would simplify the line [177](https://github.com/CanalTP/transit_model/pull/563/files#diff-48fd095c2d84a71393433f8389b49d0cR177) to something like

```rust
self.generate_quay_accessibility(stop_point.equipment_id.as_ref())
```

Reply:
Done in 3218f5a155364b1bb4924be899e08b82caa2cb27